tests/specs/tests.spec.ts

```diff
@@ -230,6 +230,36 @@ describe('tests', () => {
         ).toMatchSnapshot();
     }, 1800000);
 
+    it('consumer should produce to deadLetterTopic topic when value is not valid JSON', async () => {
+        const deadLetterTopic = 'deadLetterTopic333';
+        const topic = 'foo89';
+        await start([topic, deadLetterTopic], [{topic, targetPath: '/consume'}], {
+            DEAD_LETTER_TOPIC: deadLetterTopic,
+        });
+
+        const consumerMapping = await mockHttpTarget('/consume', 500);
+
+        await producer.send({topic, messages: [{value: '', key: 'thekey'}]});
+
+        // because we need Hamsa Hamsa Hamsa for tests to work
```

Comment:
please delete the comment 🤦🏻

Reply:
No. It will make the tests fails. I am not messing with this kind of things.